src/pip/_internal/network/auth.py

```diff
@@ -178,7 +180,7 @@ def _get_url_and_credentials(self, original_url):
             username, password = self._get_new_credentials(original_url)
 
         if username is not None or password is not None:
-            # Convert the username and password if they're None, so that
+            # Convert the username and password if they 're None, so that
```

Comment:
I think the space got its place here by mistake. You may revert this.

Reply:
Good catch, fixed in https://github.com/pypa/pip/pull/8277/commits/b5f82a0b34c2059083025431275b14a7032e3054